apis/concierge/authentication/v1alpha1/types_tls.go.tmpl

```diff
@@ -3,9 +3,24 @@
 
 package v1alpha1
 
+// CABundleSource provides a source for CA bundle used for client-side TLS verification.
+type CABundleSource struct {
+	// Whether the CA bundle is being sourced from a kubernetes secret or a configmap.
```

Comment:
We should document the Secret types that we allow.

Reply:
Yes, updated docs comment.